WordPress/gutenberg
# Add anchor support for dynamic blocks

## What?

- Adds [anchor](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-supports/#anchor) support to dynamic rendered blocks.
- Enables support for core blocks

Fixes #29401

## Testing Instructions

You need to add `'id'` to `$attributes_to_merge` in `get_block_wrapper_attributes()` from the `WP_Block_Supports` class, see https://github.com/WordPress/wordpress-develop/blob/6.0.2/src/wp-includes/class-wp-block-supports.php#L178
 ```php
$attributes_to_merge = array( 'style', 'class' );
```
Change to:
 ```php
$attributes_to_merge = array( 'style', 'class', 'id' );
```
